xcm/src/v0/traits.rs

```diff
@@ -126,14 +126,37 @@ impl Outcome {
 	}
 }
 
+/// Type of XCM message executor.
```

Comment:
Why is there both a generic and an associated `Call`?

Reply:
Removed.